src/rust/ensogl/src/display/scene.rs

```diff
@@ -933,6 +933,12 @@ impl SceneData {
             self.context.viewport(0,0,canvas.width as i32, canvas.height as i32);
         });
     }
+
+    pub fn transform_screen_to_scene_coordinates(&self, position:Vector3<f32>) -> Vector3<f32> {
```

Comment:
this can be done shorter with the same logic as `screen_to_scene_coordinates` :)